src/server/pegasus_read_service.h

```diff
@@ -23,8 +26,7 @@ class pegasus_read_service : public dsn::replication::replication_app_base,
 protected:
     // all service handlers to be implemented further
     // RPC_RRDB_RRDB_GET
-    virtual void on_get(const ::dsn::blob &args,
-                        ::dsn::rpc_replier<dsn::apps::read_response> &reply) = 0;
+    virtual void on_get(on_get_rpc rpc) = 0;
     // RPC_RRDB_RRDB_MULTI_GET
     virtual void on_multi_get(const dsn::apps::multi_get_request &args,
```

Comment:
you can refactor on_multi_get(on_sort_count、on_ttl, and so on) also, by using rpc_holder